crates/extricrate/src/lib.rs

```diff
@@ -355,15 +429,90 @@ pub mod dependencies {
                     statement_type: UseStatementType::Simple("HashMap".to_owned()),
                 }]
             );
+            assert_eq!(
+                module_b_statement.source_module,
+                "crate::module_a::module_b".into()
+            );
+            assert_eq!(module_b_statement.target_modules, vec!["foo".into()]);
+            assert_eq!(
+                module_b_statement.statement.span.start(),
+                LineColumn { line: 1, column: 0 }
+            );
+            assert_eq!(
+                module_b_statement.statement.span.end(),
+                LineColumn { line: 1, column: 8 }
+            );
+            assert_eq!(
+                module_b_statement.statement.items,
+                vec![NormalizedUseStatement {
+                    module_name: "".into(),
+                    statement_type: UseStatementType::Simple("foo".to_owned()),
```

Comment:
That's an interesting case. We might want to desugar that into:
```suggestion
                    module_name: "foo".into(),
                    statement_type: UseStatementType::Simple("self".to_owned()),
```

Or at least
```suggestion
                    module_name: "foo".into(),
                    statement_type: UseStatementType::Simple("".to_owned()),
```

Or maybe just add a TODO to that effect.

Reply:
agreed, the first option looks the neatest to me so I went with that